tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -25,15 +27,26 @@ def __init__(self, name, init_config, instances):
         self.socket_type = None
         self._addr = None
 
+        ConfigurationErrorMsg = "Invalid `{}`; a {} must be specified."
```

Comment:
```suggestion
        configuration_error_msg = "Invalid `{}`; a {} must be specified."
```

Or move to the constants section before init and put it in caps